homeassistant/components/sensor/plex.py

```diff
@@ -64,19 +75,24 @@ def __init__(self, name, plex_url, plex_user, plex_password,
         from plexapi.myplex import MyPlexAccount
         from plexapi.server import PlexServer
 
+        cert_session = None
+
         self._name = name
         self._state = 0
         self._now_playing = []
 
         if plex_token:
-            self._server = PlexServer(plex_url, plex_token)
+            self._server = PlexServer(plex_url, plex_token, cert_session)
         elif plex_user and plex_password:
             user = MyPlexAccount(plex_user, plex_password)
-            server = plex_server if plex_server else user.resources()[0].name
+            server = plex_server if plex_server \
+                else user.resources()[0].name
             self._server = user.resource(server).connect()
         else:
             self._server = PlexServer(plex_url)
 
+        _LOGGER.info("Plex Sensor Configuration done")
```

Comment:
This log isn't necessary. Home assistant core will already log when each platform is setup.